### PR TITLE
[devtools] Update DevTools Table of Content

### PR DESCRIPTION
**Pull request by jecfish**
- Add What's New in DevTools feature list
- Add DevTools Tips

### CODE REVIEW

**Comment by jecfish** [comment]
I don't understand why the TOC I added did not show up in the TOC page `/docs/devtools/`. Please help to find out.

File: site/_data/docs/devtools/toc.yml [hunk 1]
```diff
@@ -1,13 +1,11 @@
 - url: /docs/devtools/overview/
 - url: /docs/devtools/open/
+- urL: /docs/devtools/news/
```
Comment (sofiayem) [inline site/_data/docs/devtools/toc.yml:3]:
```suggestion
- url: /docs/devtools/news/
```

**Comment by sofiayem** [comment]
> I don't understand why the TOC I added did not show up in the TOC page `/docs/devtools/`. Please help to find out.

I fixed the WDNT ToC item. 'Twas a typo.

As for DTT and eng blog, my best guess is, 11ty pagination walks through files to build ToC (I think), but /tags/* is a collection. In other words, there's nesting involved that pagination can't handle.

I'd love to be able to just add link items to the ToC.

**Comment by sofiayem** [comment]
@jecfish Would workaround with a link work, wdyt?
Screenshot 2023-01-26 at 14 42 54

Technically, this icon means it's link to an external resource, but we're just linking to a tag collection on DCC.

**Review by sofiayem — APPROVED** [review]

**Review by AaronForinton — APPROVED** [review]

**Pull request merged**